packages/node_modules/@webex/internal-plugin-calendar/src/config.js

```diff
@@ -3,5 +3,8 @@
  */
 
 export default {
-  calendar: {}
+  calendar: {
```

Comment:
Please add some comments around where these configs are utilized:

```js
// Used by `syncCalendar` for the default fetch options
```

Reply:
not used anymore